### PR TITLE
file & link edits for R @ GW workshop

### PR DESCRIPTION
**Pull request by aappling-usgs**

### CODE REVIEW

File: content/course-specific-material/introR-NatGW-Aug16.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-menu: 
-  main:
-    parent: Course Specific Material
-    weight: 1
+categories: Data Science
+ 
+tags: 
+  - R
+ 
+
```
Comment (aappling-usgs) [inline content/course-specific-material/introR-NatGW-Aug16.md]: @lindsaycarr do the changes above this line look right to you? i didn't edit anything in this part of the Rmd. maybe this got autopopulated with the USGSmarkdowntemplates::hugo?

Comment (lindsaycarr) [inline content/course-specific-material/introR-NatGW-Aug16.md]: Yeah, the workflow here is kind of strange. Delete all of the lines that got added (empty author, draft: TRUE, categories, and tags). Then add the menu: stuff back in. That's needed for the collapsible menu on the sidebar

Comment (lindsaycarr) [inline content/course-specific-material/introR-NatGW-Aug16.md]: I normally copy the header, then knit, and then paste the original

Comment (ldecicco-USGS) [inline content/course-specific-material/introR-NatGW-Aug16.md]: The `USGSmarkdowntemplates` is streamlined for the blogs. If updating the training becomes a fairly common task, we can either fiddle with that package, or make a new fork of it for the training stuff.

Comment (aappling-usgs) [inline content/course-specific-material/introR-NatGW-Aug16.md]: Sounds like a good plan! I followed your directions correctly this time, right? So far the dev website looks good to me.

**Comment by lindsaycarr** [comment]
:+1: looks good

**Pull request merged**